src/main/java/uk/gov/hmcts/ccd/data/user/DefaultUserRepository.java

```diff
@@ -136,7 +136,7 @@ public UserDefault getUserDefaultSettings(final String userId) {
                 .getBody();
 
             if (userDefaultCollection == null || isEmpty(userDefaultCollection.getUserDefaults())) {
-                LOG.error("User Profile not exists for userId '{}'", userId);
+                LOG.debug("User Profile not exists for userId '{}'", userId);
```

Comment:
For the last couple of years the majority of users do NOT have a user profile. It's not an error and implementation handles properly when none is provided, so this log shouldn't be required.